src/messages.rs

```diff
@@ -64,6 +64,20 @@ where
     write_all(stream, auth_ok).await
 }
 
+/// Tell the client to use clearr text auth
+pub async fn clear_text_challenge<S>(stream: &mut S) -> Result<(), Error>
```

Comment:
Is this referenced anywhere?

Reply:
Apologies, this was another LDAP thing that I forgot to remove. This is  removed now,